templates/mysql/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,42 @@
+version: '2'
+services:
+  mysql-lb:
+    image: rancher/lb-service-haproxy
+    links:
+      - mysql
+    ports:
+      - ${mysql_lb_port}:3306
```

Comment:
Please, use lb-service-haproxy version.
Link to the service is not needed anymore.
Port definition.
```
-    image: rancher/lb-service-haproxy
+    image: rancher/lb-service-haproxy:v0.6.4
-    links:
-      - mysql
    ports:
-      - ${mysql_lb_port}:3306
+      - ${mysql_lb_port}:${mysql_lb_port}
```